apps/location_service/lib/aws_location/aws_location.ex

```diff
@@ -23,4 +23,35 @@ defmodule AWSLocation do
     Request.new([latitude, longitude])
     |> Result.handle_response([latitude, longitude])
   end
+
+  @spec autocomplete(String.t(), number) :: LocationService.Suggestion.result()
+  def autocomplete(search, limit) do
+    case ExAws.request(%ExAws.Operation.RestQuery{
```

Comment:
I didn't end up using the `Request`/`Result` infrastructure here, since I found it a bit confusing to work with. If there are any ideas on how to extend / refactor that to make it applicable here I'm definitely open to it.

Reply:
This should be good now 👍